parsec/mca/device/device.h

```diff
@@ -143,6 +143,7 @@ struct parsec_device_module_s {
     int64_t   gflops_fp32;  /**< Number of single precision operations per nanosecond (or gflops/s) */
     int64_t   gflops_fp64;  /**< Number of double precision operations per nanosecond (or gflops/s) */
     int64_t   gflops_tf32;  /**< Number of tensor operations per nanosecond (or gflops/s) */
+    uint8_t   gflops_guess; /**< True if the device is not 'known' which entails that the 'gflops' rates have been populated with fallback (arbitrary) values. */
```

Comment:
Move this at the end of the structure to allow the compiler to pack it more efficiently.

Reply:
This is a good point to move it to: https://github.com/ICLDisco/parsec/pull/577/files#diff-3a7133ac2f9430b1fc1cde3edef7f3b7b6d060e86ec881967ae51f585a781c21R142

Reply:
The link does not work for me, but I am referring to the end of the structure where the 3 other uint8_t fields are located.